src/components/modal/modalsContainer.ts

```diff
@@ -0,0 +1,21 @@
+import Block from '../../scripts/block/block';
+import { Props } from '../../scripts/dto/types';
+
+export default class ModalsContainer extends Block {
+  constructor(props: Props) {
+    super({
+      tagName: props.tag || 'div'
+    }, 'div(data-child="modals notifications", class=\`modal-bg \${classList ? classList : ""}\`)', {
+      events: {
+        click: (e: Event) => {
+          const target: HTMLElement = e.target as HTMLElement;
```

Comment:
```suggestion
          const target = e.target as HTMLElement;
```
Можно оставить в таком виде